configs/frontend_config/config.json

```diff
@@ -1 +1,28 @@
-{}
\ No newline at end of file
+{
+  "@openmrs/esm-patient-chart-app": {
+    "extensionSlots": {
+      "patient-chart-dashboard-slot": {
+        "add": [
+          "nav-group#HIVCareTreatment"
+        ],
+        "configure": {
+          "nav-group#HIVCareTreatment": {
+            "title": "HIV Care and Treatment",
```

Comment:
I'd do a similar thing with title as a key and provide `Translation overrides` for this extension.